model/converter/json/to_domain.go

```diff
@@ -184,6 +185,9 @@ func (td toDomain) convertLogs(logs []json.Log) ([]model.Log, error) {
 }
 
 func (td toDomain) convertProcess(process *json.Process) (*model.Process, error) {
+	if process == nil {
```

Comment:
don't think we ever will, but if we convert UI model to domain process will be nil